PowerShellEditorServices.build.ps1

```diff
@@ -37,7 +37,7 @@ task SetupDotNet -Before Clean, Build, TestHost, TestServer, TestProtocol, TestP
     }
 
     # Make sure the dotnet we found is the right version
-    if ($dotnetExePath -and (& $dotnetExePath --version) -eq $requiredSdkVersion) {
+    if ($dotnetExePath -and [version](& $dotnetExePath --version) -ge [version]$requiredSdkVersion) {
```

Comment:
So why don't we just use a `global.json` file to specify the SDK version? See https://github.com/PowerShell/PowerShell/blob/master/global.json

Reply:
@rkeithhill that's kinda opposite of why I opened this PR. I didn't want to require 1 particular SDK version. I wanted to be able to build using the dotnet sdk of any version greater than 2.0.

Ex. I have 2.1.4 on my machine and it builds just fine.

By adding a `global.json` it forces the construct of "you need this exact version" at least until this is implemented:
https://github.com/dotnet/core-setup/issues/679

Reply:
I was just looking for a mechanism to set a **minimum** version, and lo and behold it led me right back here from that core-setup issue

Reply:
It's so sad. I just want to not have to download a million dotnet sdks 😭

Reply:
I don't know if it is dl of dotnet or what but every time I debug the extension, it builds PSES and that part is **sloooow**. :-(

Reply:
That's because debugging the extension runs `Invoke-Build BuildAll ` I believe.

Reply:
But that's in the vscode-powershell repo. dotnet is only downloaded once.

Reply:
Is that a "rebuild" or an incremental build?  You'd think that build process would be faster given that I've usually not changed any PSES source.

Reply:
Do you guys have the "Launch Extension (Build client only)" option? I haven't tried it, but would assume it avoids rebuilding PSES.

Reply:
But adding automatic detection would be nice. Not sure if that's a dotnet-build capability.

Reply:
dotnet build times have supposedly improved noticeably in the 2.1 preview.  Maybe I should try that version to see how it performs.

Reply:
dotnet 2.1 seems good to me :)

this PR will allow you to use that build if you have it on your machine already... but still be compat with 2.0 if folks have that

Reply:
@rkeithhill is that thumbs up a sign off 😉

Reply:
BTW there are a bit too many warnings IMO.  At some point, we should clean up what we can. I see a couple of variable not used warnings that should be easy to fix.

Reply:
Agreed. The unused variable warnings would make a great Up-for-grabs I think.